client/core/core.go

```diff
@@ -2554,6 +2558,61 @@ func handleTradeSuspensionMsg(c *Core, dc *dexConnection, msg *msgjson.Message)
 	return nil
 }
 
+// handleTradeResumptionMsg is called when a trade resumption notification is received.
+func handleTradeResumptionMsg(c *Core, dc *dexConnection, msg *msgjson.Message) error {
+	var rs msgjson.TradeResumption
+	err := msg.Unmarshal(&rs)
+	if err != nil {
+		return fmt.Errorf("trade resumption unmarshal error: %v", err)
+	}
+
+	dc.pendingResumesMtx.Lock()
+	defer dc.pendingResumesMtx.Unlock()
+
+	// Ensure the provided market exists for the dex.
+	dc.marketMtx.RLock()
+	mkt, ok := dc.marketMap[rs.MarketID]
+	dc.marketMtx.RUnlock()
+	if !ok {
+		return fmt.Errorf("no market found with ID %s", rs.MarketID)
+	}
+
+	// Ensure the market is suspended.
+	if !mkt.Suspended {
+		return fmt.Errorf("market %s for dex %s is not suspended",
+			rs.MarketID, dc.acct.host)
+	}
+
+	// Remove pending resumes for the market.
+	if sched := dc.pendingResumes[rs.MarketID]; sched != nil {
+		sched.Stop()
+		delete(dc.pendingResumes, rs.MarketID)
+	}
+
+	// TODO: the server on sending the resumption message would know the exact
+	// resumption time, perhaps the message should send that instead of the
+	// EpochLen here?
+	resumeTime := rs.StartEpoch * rs.EpochLen
+
+	// Set the new scheduled resume.
+	duration := time.Until(encode.UnixTimeMilli(int64(resumeTime)))
+	dc.pendingSuspends[rs.MarketID] = time.AfterFunc(duration, func() {
+		// Update the market as resumed.
+		err := dc.resume(rs.MarketID)
```

Comment:
I think part of actually doing the resume it to get the latest 'config' and deal with any changes.